### PR TITLE
Prevent from making ip shared if it's not owned by admin

### PR DESCRIPTION
**Pull request by cmstew**
Since users can't share ip's there's no point in allowing the option to even be set.

### CODE REVIEW

File: bin/v-change-sys-ip-owner [hunk 1]
```diff
@@ -31,6 +31,12 @@ is_ip_valid "$ip"
 is_ip_key_empty '$U_WEB_DOMAINS'
 is_ip_key_empty '$U_SYS_USERS'
 
+ip_status="$(get_ip_value '$STATUS')"
+
+if [ "$user" != "admin" ] && [ "$ip_status" == "shared" ]; then
+        check_result "$E_INVALID" "Only the 'admin' user can have a shared ip"
```
Comment (Lupul) [inline bin/v-change-sys-ip-owner]: Correcting a previous invalid status with `ip_status="dedicated"` here would be better than aborting

File: bin/v-change-sys-ip-status [hunk 2]
```diff
@@ -35,6 +35,9 @@ ip_owner=$(get_ip_value '$OWNER')
 if [ "$web_domains" -ne '0' ] && [ "$sys_user" != "$ip_owner" ]; then
     check_result "$E_INUSE" "ip $ip is used"
 fi
+if [ "$ip_owner" != "admin" ]; then
```
Comment (Lupul) [inline bin/v-change-sys-ip-status]: `if [ "$ip_owner" != "admin" ] && [ "$ip_status"="shared" ]; then`
So it won't abort when switching ipstatus from shared to dedicated on ips incorrectly configured as shared previously

**Review by Lupul — CHANGES_REQUESTED** [review]
PR is good just needs minor tweaks for backwards compatibility

File: bin/v-change-sys-ip-owner (hunk 1 above)
Comment (cmstew) [reply bin/v-change-sys-ip-owner]: should be good now.

File: bin/v-change-sys-ip-status (hunk 2 above)
Comment (cmstew) [reply bin/v-change-sys-ip-status]: should be good now.

**Pull request merged**